.github/pull_request_template.md

```diff
@@ -4,7 +4,7 @@ See:
 
 **Checklist**
 
-* [ ] Write/update ITs
```

Comment:
Good! We could also drop the `* ... links to JIRA tickets ...` section, it doesn't make sense anymore with the tickets that are automatically linked from the commits.